tests/language/nnbd/normalization/generic_function_type_object_normalization_test.dart

```diff
@@ -51,7 +51,7 @@ class SimpleBoundTests<T> {
   T h2<R extends FutureOr<T>>() => throw "Unused";
 
   // If `T` is a top type, check that the appropriate
-  // equalities and disequalities hold.
+  // equalities and inequalities hold.
```

Comment:
notable

Reply:
> notable

Not sure what that means. Anyway, this one is tricky, because 'inequality' commonly refers to relations like `<` and `>=`, and in this context the topic is `==` and `!=`. I'd recommend that we keep 'disequalities'.

Reply:
I use `notable` to call out potentially nontrivial/objectionable changes. Things where I think I'm right but, as here, could be wrong if I don't properly understand the domain.

I'll drop this and fix the case of async shortly.

Reply:
Sounds good, thanks!